.travis.yml

```diff
@@ -1,21 +1,31 @@
 sudo: false
 language: python
-python: 3.6
+python: 3.7
 cache: pip
 fast_finish: true
+addons:
+  apt_packages:
+    - pandoc
 env:
   global:
     - buildnum=$TRAVIS_BUILD_NUMBER
 install:
   - pip install pre-commit
```

Comment:
I think this is unnecessary now that "pre-commit" has been added to requirements.txt.

Reply:
removed.